pkg/doc/verifiable/credential.go

```diff
@@ -697,6 +697,34 @@ func NewCredential(vcData []byte, opts ...CredentialOpt) (*Credential, []byte, e
 	return vc, vcDataDecoded, nil
 }
 
+// NewUnverifiedCredential decodes Verifiable Credential from bytes which could be marshalled JSON or serialized JWT.
+// It does not make a proof check though. Can be used for purposes of decoding of VC stored in a wallet.
+// Please use this function with caution.
+func NewUnverifiedCredential(vcBytes []byte) (*Credential, error) {
```

Comment:
Note: this is a `Parse` method that doesn't validate.
TODO: review method namings. And options strategy.

(I was also tempted to ask if it is an `Unsafe` method).

Reply:
Yeah, when I named it I recalled our today's call and the naming strategy in regards to `Parse()` ([]byte) and `Decode()` (struct).
I named it as `NewUnverifiedCredential()` in order to be consistent with already present `NewCredential()`. I agree that the naming should be revisited in `verifiable`. I remember we wanted to make the package refactoring after closing major dev issues.
Also, I named it "Unverifiable" to emphasize the contrast to the "verifiable" nature of Credentials.